OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -134,6 +134,7 @@ public interface INotifyProduction { void UnitProduced(Actor self, Actor other,
 	public interface INotifyOtherProduction { void UnitProducedByOther(Actor self, Actor producer, Actor produced, string productionType, TypeDictionary init); }
 	public interface INotifyDelivery { void IncomingDelivery(Actor self); void Delivered(Actor self); }
 	public interface INotifyDocking { void Docked(Actor self, Actor harvester); void Undocked(Actor self, Actor harvester); }
+	public interface INotifyResourceDumped { void Dumped(Actor self, int amount); }
```

Comment:
Please change this to
```csharp
public interface INotifyResourceAccepted { void OnResourceAccepted(Actor self, Actor refinery, int amount); }
```
to better match the `IAcceptResources` interface and to avoid confusion as to which actor this is being called with/for.